_inc/client/plans/single-product-components/purchased-product-card.jsx

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
Huh, it is like you went in here to actually improve the code rather than just hacking something in... :)

Reply:
😉